internal/resolvers/frankerfacez/resolver.go

```diff
@@ -37,7 +38,7 @@ var (
 	errInvalidFrankerFaceZEmotePath = errors.New("invalid FrankerFaceZ emote path")
 )
 
-func New() (resolvers []resolver.CustomURLManager) {
+func New(cfg config.APIConfig) (resolvers []resolver.CustomURLManager) {
```

Comment:
I'm perfectly fine with this solution - where we pass through the config object even if it's not used by a resolver